simde/x86/sse4.2.h

```diff
@@ -241,6 +241,57 @@ simde_mm_cmpistrz_16_(simde__m128i b) {
   #define _mm_cmpistrz(a, b, imm8) simde_mm_cmpistrz(a, b, imm8)
 #endif
 
+SIMDE_FUNCTION_ATTRIBUTES
+int64_t bitreflection(int64_t n) {
```

Comment:
These should all be in the simde namespace.  They're intended to be private APIs, so use a trailing underscore (e.g., `simde_mm_crc32_bitreverse_`, `simde_mm_crc32_mod2div_`).

Usually you could just make these static functions in a C file or a private header, or even just make them normal symbols but set the [visibility](https://gcc.gnu.org/wiki/Visibility) to hidden.  However, since SIMDe is a header-only library, we don't really have a way to hide implementations.